hts-precompile/IHederaTokenService.sol

```diff
@@ -441,6 +568,181 @@ interface IHederaTokenService {
     /// @param operator The address that acts on behalf of the owner
     /// @return responseCode The response code for the status of the request. SUCCESS is 22.
     /// @return approved True if `operator` is an approved operator for `owner`, false otherwise
-    function isApprovedForAll(address token, address owner, address operator)  external returns (int responseCode, bool approved);
+    function isApprovedForAll(
+        address token,
+        address owner,
+        address operator
+    ) external returns (int responseCode, bool approved);
+
+    /// Query if token account is frozen
+    /// @param token The token address to check
+    /// @param account The account address associated with the token
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    /// @return frozen True if `account` is frozen for `token`
+    function isFrozen(address token, address account)
+        external
+        returns (int responseCode, bool frozen);
+
+    /// Query if token account has kyc granted
+    /// @param token The token address to check
+    /// @param account The account address associated with the token
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    /// @return kycGranted True if `account` has kyc granted for `token`
+    function isKyc(address token, address account)
+        external
+        returns (int responseCode, bool kycGranted);
+
+    /// Operation to delete token
+    /// @param token The token address to be deleted
+    /// @return responseCode The response code for the status of the request. SUCCESS is 22.
+    function deleteToken(address token) external returns (int responseCode);
+
+    /// Query token custom fees
+    /// @param token The token address to check
+    /// @return customFees Set of custom fees for `token`
+    function getTokenCustomFees(address token)
+        external
+        returns (CustomFee[] memory customFees);
```

Comment:
Should there also be a responseCode returned here to be inline with the other function definitions?